release-management.md

```diff
@@ -0,0 +1,148 @@
+  - Once no triaged issues or prs are scoped to the release, an rc is cut.
+  - To minimize confusion, and the risk of an alpha being widely deployed, alpha branches and released binaries may be removed at the discretion, and a [two-thirds supermajority][super-majority] vote, of the maintainers.
+Maintainers will create an Issue, and vote upon it for transparency to the decision to remove a release and/or branch.
+  - Not supported
+- **rc.n:** `X.Y.Z-rc.n`
+  - Released as needed before a final version is released
+  - Bugfixes on new features only as reported through usage
+  - An rc is not expected to revert to an alpha release.
+  - Once no triaged issues or prs are scoped to the release, an final version is cut.
+  - A release candidate will typically have at least two weeks of bake time, providing the community time to provide feedback.
+  - Release candidate are cut from the branch where the work is done.
+  - To minimize confusion, and the risk of an rc being widely deployed, rc branches and released binaries may be removed at the discretion, and a [two-thirds supermajority][super-majority] vote, of the maintainers.
+Maintainers will create an Issue, and vote upon it for transparency to the decision to remove a release and/or branch.
+  - Not supported
+
+## Supported Releases
+
+The go-cose maintainers expect to "support" n (current) and n-1 major.minor releases.
+"Support" means we expect users to be running that version in production.
+For example, when v1.3.0 comes out, v1.1.x will no longer be supported for patches, and the maintainers encourage users to upgrade to a supported version as soon as possible.
+Support will be provided best effort by the maintainers via GitHub issues and pull requests from the community.
+
+The go-cose maintainers expect users to stay up-to-date with the versions of go-cose release they use in production, but understand that it may take time to upgrade.
+We expect users to be running approximately the latest patch release of a given minor release and encourage users to upgrade as soon as possible.
+
+While pre-releases may be deleted at the discretion of the maintainers, all Major, Minor and Patch releases should be maintained.
+Only in extreme circumstances, as agreed upon by a [two-thirds supermajority][super-majority] of the maintainers, shall a release be removed.
+
+Applicable fixes, including security fixes, may be cherry-picked into the release branch, depending on severity and feasibility.
+Patch releases are cut from that branch as needed.
+
+## Security Reviews
+
+The go-cose library is an sdk around underlying crypto libraries, tailored to COSE scenarios.
+The go-cose library does not implement cryptographic functionality, reducing the potential risk.
+To assure go-cose had the proper baseline, two [security reviews](./reports) were conducted prior to the [v1.0.0](https://github.com/veraison/go-cose/releases/tag/v1.0.0) release
```

Comment:
```suggestion
```

Reply:
No need for history in a policy doc

Reply:
The purpose was to reference the existing security reviews, to provide visibility and confidence in how we manage our policy. We have, and will do security reviews, but will not do them for all releases.
How do we create visibility to reviews, so it doesn't look like we're dismissing the need?